mappings/net/minecraft/client/gui/DrawableHelper.mapping

```diff
@@ -23,6 +23,10 @@ CLASS net/minecraft/class_332 net/minecraft/client/gui/DrawableHelper
 		ARG 7 texHeight
 		ARG 8 texWidth
 	METHOD method_25292 hLine (IIII)V
+		ARG 1 x1
+		ARG 2 x2
+		ARG 3 y
+		ARG 4 color
 	METHOD method_25293 blit (IIIIFFIIII)V
```

Comment:
This definitely needs to be renamed, though maybe in another pr.

Reply:
I am not sure how I can rename this, there are just too many `blit`s, probably with another pr. my suggestion would be `drawTexture` because I can't think of better names.

Reply:
I might just rename `blitOffset` to `zLevel` for now to be consistent to ItemRenderer?